frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/QuestionDetails.tsx

```diff
@@ -45,9 +51,20 @@ export const QuestionDetails = ({ question }: { question: Question }) => {
         </Flex>
       </SidesheetCardSection>
       <SidesheetCardSection title={t`Saved in`}>
-        <Flex gap="sm" align="center">
-          <Icon name="folder" />
-          <Text>{question.collection()?.name}</Text>
+        <Flex gap="sm" align="top" color="var(--mb-color-brand)">
+          <Icon
+            name="folder"
+            color="var(--mb-color-brand)"
+            className={SidebarStyles.IconMargin}
+          />
+          <Text>
+            <Link
+              to={`/collection/${question.collection()?.id}`}
+              variant="brand"
+            >
+              {question.collection()?.name}
```

Comment:
If `question.collection()` can be `undefined`, why isn't the "Saved in" section displayed conditionally? If we know that `question.collection()` is defined, perhaps using typescript's `as` is a more conventional way to make that knowledge clear.